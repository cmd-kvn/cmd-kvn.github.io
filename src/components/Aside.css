aside {
    width: 33%;
    height: 100%;
    float: left;
}

img {
    width: 100%
}