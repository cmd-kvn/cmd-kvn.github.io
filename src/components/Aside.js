import React from 'react';

import './Aside.css';

const Aside = () => {
    return (
        <aside>
            <img src='headshot.JPG' alt='kevin headshot' />
        </aside>
    );
};

export default Aside;
