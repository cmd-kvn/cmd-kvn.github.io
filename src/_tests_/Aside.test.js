/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Aside from '../components/Aside';

describe('Aside', function () {
    it('runs the Aside', function () {
        const rendered = shallow(<Aside />);
        expect(toJson(rendered)).toMatchSnapshot();
    })
});