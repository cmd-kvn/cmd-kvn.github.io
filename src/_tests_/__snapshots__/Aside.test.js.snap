exports[`Aside runs the Aside 1`] = `
<aside>
  <img
    alt="kevin headshot"
    src="headshot.JPG" />
</aside>
`;
